libs/books/data-access/src/lib/+state/reading-list.actions.ts

```diff
@@ -32,6 +33,21 @@ export const removeFromReadingList = createAction(
   props<{ item: ReadingListItem }>()
 );
 
+export const markedAsFinished = createAction(
+  '[Reading List] Marked as finished',
+  props<{ item: ReadingListItem }>()
+);
+
+export const markFinshedSuccess = createAction(
```

Comment:
I would keep the naming of the action consistent with the other "success" marking actions (eg. `confirmedMarkAsFinished`). Same goes for the action text, `[Reading List] Confirmed mark as finished`.